suites/gcp.xml

```diff
@@ -0,0 +1,170 @@
+        <classes>
+            <class name="io.managed.services.test.devexp.KafkaOperatorTest"/>
+        </classes>
+    </test> 
+    <test name="ServiceRegistryOperatorTest">
+        <classes>
+            <class name="io.managed.services.test.devexp.ServiceRegistryOperatorTest"/>
+        </classes>
+    </test> 
+    <test name="KafkaAccessMgmtTest">
+        <classes>
+            <class name="io.managed.services.test.kafka.KafkaAccessMgmtTest"/>
+        </classes>
+    </test>
+    <test name="KafkaInstanceAPITest">
+        <classes>
+            <class name="io.managed.services.test.kafka.KafkaInstanceAPITest"/>
+        </classes>
+    </test>
+    <test name="KafkaMgmtAPITest">
+        <classes>
+            <class name="io.managed.services.test.kafka.KafkaMgmtAPITest"/>
+        </classes>
+    </test>
+    <test name="KafkaOverseeingTest">
+        <classes>
+            <class name="io.managed.services.test.kafka.KafkaOverseeingTest"/>
+        </classes>
+    </test>
+    <test name="KafkaRegressionTest">
+        <classes>
+            <class name="io.managed.services.test.kafka.KafkaRegressionTest"/>
+        </classes>
+    </test>
+    <test name="LongLiveKafkaInstanceTest">
+        <classes>
+            <class name="io.managed.services.test.kafka.LongLiveKafkaInstanceTest"/>
+        </classes>
+    </test>
+    <test name="QuotaKafkaInstanceTest">
+        <classes>
+            <class name="io.managed.services.test.kafka.QuotaKafkaInstanceTest"/>
+        </classes>
+    </test>
+
+    <test name="KafkaInstanceContext">
+        <classes>
+            <class name="io.managed.services.test.quickstarts.contexts.KafkaInstanceContext"/>
+        </classes>
+    </test>
+    <test name="OpenShiftAPIContext">
+        <classes>
+            <class name="io.managed.services.test.quickstarts.contexts.OpenShiftAPIContext"/>
+        </classes>
+    </test>
+    <test name="ServiceAccountContext">
+        <classes>
+            <class name="io.managed.services.test.quickstarts.contexts.ServiceAccountContext"/>
+        </classes>
+    </test>
+    <test name="UserContext">
+        <classes>
+            <class name="io.managed.services.test.quickstarts.contexts.UserContext"/>
+        </classes>
+    </test>
+    <test name="KafkaAccessSteps">
+        <classes>
+            <class name="io.managed.services.test.quickstarts.steps.KafkaAccessSteps"/>
+        </classes>
+    </test>
+    <test name="KafkaInstanceSteps">
+        <classes>
+            <class name="io.managed.services.test.quickstarts.steps.KafkaInstanceSteps"/>
+        </classes>
+    </test>
+    <test name="KafkaScriptsSteps">
+        <classes>
+            <class name="io.managed.services.test.quickstarts.steps.KafkaScriptsSteps"/>
+        </classes>
+    </test>
+    <test name="KafkaTopicSteps">
+        <classes>
+            <class name="io.managed.services.test.quickstarts.steps.KafkaTopicSteps"/>
+        </classes>
+    </test>
```

Comment:
there are no tests in these context, why are they here ?  (same for those below)

Reply:
Removed all test with .contexts and .steps